images/scripts/opensuse-tumbleweed.setup

```diff
@@ -117,12 +120,17 @@ su builder -c "sudo mock --verbose -i $(/var/lib/testvm/build-deps.sh "${ID} ${V
 
 # Prevent SSH from hanging for a long time when no external network access
 echo 'UseDNS no' >> /etc/ssh/sshd_config
+# Pam is not enabled by default and is needed for systemctl --user to work
+echo 'UsePAM yes' >> /etc/ssh/sshd_config
 
 # Audit events to the journal
 rm -f '/etc/systemd/system/multi-user.target.wants/auditd.service'
 rm -rf /var/log/audit/
 
 systemctl enable --now systemd-networkd.service
+# dbus-broker needs to be manually enabled on opensuse-tumbleweed
+systemctl enable --now dbus-broker.service
+systemctl --global enable dbus-broker.service
```

Comment:
:eyes: There's no running D-Bus daemon on openSUSE by default? That is by far the most worrying thing of all then. Or is that "you specifically want dbus-broker instead of dbus-daemon"? If dbus-daemon is the default, then let's please use that.

Reply:
The current image *does* have a running `dbus.service` (`dbus-daemon`), so let's please keep that. That feels like a distro choice, not something which we should change in our tests.

Reply:
There's D-Bus daemon by default, so this is "I prefer to use dbus-broker". But you're right. As dbus-daemon is the default. I should figure out a way to use it instead.

Reply:
> I should figure out a way to use it instead.

At least the system one does out of the box. I tried to do the `echo 'UsePAM yes' >> /etc/ssh/sshd_config` and I can do `busctl --user` -- `systemctl --user status dbus.socket` is running, so it spawns it on demand. That seems "good enough", and what the distro decided to do. So what needs fixing?